dsc/src/util.rs

```diff
@@ -55,6 +57,23 @@ pub const EXIT_DSC_RESOURCE_NOT_FOUND: i32 = 7;
 pub const DSC_CONFIG_ROOT: &str = "DSC_CONFIG_ROOT";
 pub const DSC_TRACE_LEVEL: &str = "DSC_TRACE_LEVEL";
 
+#[derive(Deserialize)]
+pub struct TracingSetting {
+    level:  TraceLevel,
```

Comment:
Use `///` comments to describe the members

Reply:
updated.